packages/gatsby-source-contentful/src/__tests__/form-generation.js

```diff
@@ -0,0 +1,45 @@
+import { pluginOptionsSchema } from "../gatsby-node"
+import table from "markdown-table"
+
+const generateMarkdownTable = schema => {
```

Comment:
Why is this in the test file?

Reply:
I'm guessing just for trying-out purposes, but I don't see any reason it can't be an in exported file already, to be clearer

Reply:
Not in any way an important choice! Sorry for the confusion, will move it to its own file.